ouroboros-network/src/Ouroboros/Network/Diffusion/Configuration.hs

```diff
@@ -0,0 +1,109 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Ouroboros.Network.Diffusion.Configuration
+  ( module Ouroboros.Network.Diffusion.Configuration
+    -- re-exports
+  , AcceptedConnectionsLimit (..)
+  , BlockFetchConfiguration (..)
+  , ChainSyncTimeout (..)
+  , MiniProtocolParameters (..)
+  , P2P (..)
+  , PeerSelectionTargets (..)
+  , PeerSharing (..)
+  , defaultMiniProtocolParameters
+  , deactivateTimeout
+  , closeConnectionTimeout
+  , peerMetricsConfiguration
+  , defaultTimeWaitTimeout
+  , defaultProtocolIdleTimeout
+  , defaultResetTimeout
+  , handshake_QUERY_SHUTDOWN_DELAY
+  , ps_POLICY_PEER_SHARE_STICKY_TIME
+  , ps_POLICY_PEER_SHARE_MAX_PEERS
+  ) where
+
+import System.Random (randomRIO)
+
+import Ouroboros.Network.BlockFetch (BlockFetchConfiguration (..))
+import Ouroboros.Network.ConnectionManager.Core (defaultProtocolIdleTimeout,
+           defaultResetTimeout, defaultTimeWaitTimeout)
+import Ouroboros.Network.Diffusion (P2P (..))
+import Ouroboros.Network.Diffusion.Policies (closeConnectionTimeout,
+           deactivateTimeout, peerMetricsConfiguration)
+import Ouroboros.Network.NodeToNode (MiniProtocolParameters (..),
+           defaultMiniProtocolParameters)
+import Ouroboros.Network.PeerSelection.Governor.Types
+           (PeerSelectionTargets (..))
+import Ouroboros.Network.PeerSelection.PeerSharing (PeerSharing (..))
+import Ouroboros.Network.PeerSharing (ps_POLICY_PEER_SHARE_MAX_PEERS,
+           ps_POLICY_PEER_SHARE_STICKY_TIME)
+import Ouroboros.Network.Protocol.ChainSync.Codec (ChainSyncTimeout (..))
+import Ouroboros.Network.Protocol.Handshake (handshake_QUERY_SHUTDOWN_DELAY)
+import Ouroboros.Network.Protocol.Limits (shortWait)
+import Ouroboros.Network.Server.RateLimiting (AcceptedConnectionsLimit (..))
+
+
+newtype DefaultQtyBootstrapPeers = DefaultQtyBootstrapPeers { getDefaultQtyBootstrapPeers :: Int }
+  deriving (Eq, Show)
```

Comment:
hmm, what `Qty` stands for?

Reply:
Does this comes from the `targetNumberOfBootstrapPeers` if so, this is no longer needed as a configuration.

Reply:
It's an established abbreviation for quantity, the type was getting a little long with Number and I thought this was a good synonym. Should I change to Number or Count?

Reply:
what's this config about?

Reply:
Based on my discussion with @coot we want to keep this constant around for some time even after transitioning to genesis but it will go away at some point.